components/fieldgroups/experience-event/experienceevent-campaign-delivery-log.schema.json

```diff
@@ -0,0 +1,174 @@
+            "LINE Channel": "LINE",
+            "Other": "Other",
+            "Undefined": "Not Defined"
+          },
+          "default": "Undefined"
+        },
+        "xdm:deliveryName": {
+          "title": "Delivery Name",
+          "type": "string",
+          "description": "Internal Name that uniquely identifies the delivery."
+        },
+        "xdm:deliveryLabel": {
+          "title": "Delivery Label",
+          "type": "string",
+          "description": "A human-friendly identifier of the delivery which is originating this message."
+        },
+        "xdm:deliveryMode": {
+          "title": "Delivery Mode",
+          "type": "string",
+          "description": "Mode of delivery for the messages sent."
+        },
+        "xdm:quarantine": {
+          "title": "Quarantine",
+          "type": "string",
+          "description": "Address moved to quarantine. The quarantine is set to `true` when the message failed to deliver in a way that indicates that no future messages will be deliverable either to this address."
+        },
+        "xdm:controlGroupMember": {
+          "title": "Control Group Member",
+          "type": "string",
+          "description": "The recipient of this message is a member of a control group for this campaign activity. A control group is a population which will not receive the delivery; it is used to track post-delivery behavior and campaign impact by making a comparison between the control group (or hold out group) with the behavior of target population, which has received the delivery."
+        },
+        "xdm:seedMember": {
+          "title": "Seed Member",
+          "type": "string",
+          "description": "The recipient of this message is a member of a seed list for this campaign activity. Seed addresses are used to include recipients who do not match the defined target criteria. Seed members could be your marketing team or other stakeholders who are interested in tracking/measuring the quality of the campaign. SeedMembers get the actual message, at the same time with regular recipients, even when the seedMember does not qualify to be a regular recipient."
+        },
+        "xdm:proofMember": {
+          "title": "Proof Member",
+          "type": "string",
+          "description": "The recipient of this message is a member of the proof group for this campaign activity. The proof is a special message that lets you test a delivery before sending to the main target. Proof recipients are responsible for approving both the form and content of the message. It is a pre-delivery target."
+        },
+        "xdm:status": {
+          "title": "Message Status",
+          "type": "string",
+          "description": "Indicates status of the message that is being delivered."
+        },
+        "xdm:reason": {
+          "title": "Reason",
+          "type": "string",
+          "description": "The reason why the message could/would not be delivered.",
+          "meta:enum": {
```

Comment:
Note this meta:enum is not an actual enum declaration but only a way to define a friendly name to actual values that would exist. If you want this field actually constrained to a set of specific values only then you need a "enum" attribute. As it is here the field will accept any random string.

Reply:
I agree.  However, the reason to not put the constraints is if a new enum value gets added in the source this field group needs to be modified to avoid any failures.  While this is do-able the chances of failures are still high if customer/engineering team forgets to modify the field group.  Enumerations are maintained by the customer and though it is a rare change it can happen.